externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -181,22 +182,43 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
     } catch {
       case e: KyuubiSQLException =>
         warn(s"Error closing session ${sessionHandle}", e)
+    } finally {
+      val sessionSavePath = getSessionResultSavePath(sessionHandle)
+      try {
+        val path = new Path(sessionSavePath)
+        val fs = path.getFileSystem(spark.sparkContext.hadoopConfiguration)
```

Comment:
Consider the case of `OPERATION_RESULT_SAVE_TO_FILE=false` to avoid accessing fs.

Reply:
adopted